filebeat/harvester/harvester.go

```diff
@@ -17,12 +17,15 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/satori/go.uuid"
+
 	"github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester/encoding"
 	"github.com/elastic/beats/filebeat/input"
 )
 
 type Harvester struct {
+	Id                 uuid.UUID
```

Comment:
Yeah, I think it makes sense to introduce the Havester ID in #964. We can discuss more there. If we need the beat UUID we can (preferable) inject it or have a singleton that makes it available.